android/app/src/main/java/com/couchbase/todo/db/DAO.java

```diff
@@ -282,6 +288,27 @@ public void deleteById(@NonNull String docId) {
         catch (CouchbaseLiteException e) { reportError(e); }
     }
 
+    public void logAll() throws CouchbaseLiteException {
+        Query query = createQuery(
+                SelectResult.all());
+        ResultSet results = query.execute();
+        for(Result result:results){
+            Log.i("allData", result.toJSON());
+        }
+    }
+
+    public ResultSet getAllTasksFromList(String listID) throws CouchbaseLiteException {
+       Query query= createQuery(SelectResult.expression(Meta.id),
```

Comment:
Sooooo.... this isn't going to log the whole task document, right?  I think that the requirement was to log a list, or to log a task.  This might be a good thing, but it isn't on the list of things to do and you'll have to explain it to me...